autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/DistroXAutoscaleClusterResponse.java

```diff
@@ -0,0 +1,134 @@
+package com.sequenceiq.periscope.api.model;
+
+import java.util.List;
+
+import com.sequenceiq.periscope.doc.ApiDescription;
+import com.sequenceiq.periscope.doc.ApiDescription.ClusterJsonsProperties;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+public class DistroXAutoscaleClusterResponse implements Json {
+    @ApiModelProperty(ClusterJsonsProperties.ID)
+    private long id;
+
+    @ApiModelProperty(ClusterJsonsProperties.STACK_NAME)
+    private String stackName;
+
+    @ApiModelProperty(ClusterJsonsProperties.STACK_CRN)
+    private String stackCrn;
+
+    @ApiModelProperty(ClusterJsonsProperties.STATE)
+    private String state;
+
+    @ApiModelProperty(ClusterJsonsProperties.STACK_TYPE)
+    private String stackType;
+
+    @ApiModelProperty(ClusterJsonsProperties.AUTOSCALING_ENABLED)
+    private Boolean autoscalingEnabled;
+
+    @ApiModelProperty(ApiDescription.DistroXClusterJsonsProperties.DISTROX_SCALING_MODE)
+    private AutoscalingModeType autoScalingMode;
+
+    @ApiModelProperty(ClusterJsonsProperties.TIME_ALERTS)
+    private List<TimeAlertResponse> timeAlerts;
+
+    @ApiModelProperty(ClusterJsonsProperties.LOAD_ALERTS)
+    private List<LoadAlertResponse> loadAlerts;
+
+    @ApiModelProperty(ClusterJsonsProperties.SCALING_CONFIGURATION)
+    private ScalingConfigurationRequest scalingConfiguration;
```

Comment:
ScalingConfigurarationResponse? (Likely identical to ScalingConfigurationRequest - but should still be a separate class), or maybe just name the object ScalingConfiguration.

Reply:
All Request and Response are identical except they have DB ID in response, the naming convention and converter conventions is based on existing class hierarchy.